mne/tests/test_docstring_parameters.py

```diff
@@ -275,19 +276,23 @@ def test_documented():
     else:
         public_modules_.append('mne.gui')
 
-    doc_file = op.abspath(op.join(op.dirname(__file__), '..', '..', 'doc',
-                                  'python_reference.rst'))
+    doc_dir = op.abspath(op.join(op.dirname(__file__), '..', '..', 'doc'))
+    api_dir = op.join(doc_dir, 'api')
+    doc_file = op.join(doc_dir, 'python_reference.rst')
```

Comment:
These changes can probably be reverted now?

Reply:
not simply reverted, as we now have to check 22 separate files instead of just `python_reference.rst`